ops/iiif.tmpl.yaml

```diff
@@ -0,0 +1,325 @@
+replicaCount: 2
```

Comment:
delete this file, its for hyku to deploy to r2-staging.

Reply:
keeping inline with Hyku